<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win</class>
 <widget class="QWidget" name="win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>661</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/sky.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="star3">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>100</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/win.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="star3_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>-50</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/win.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="star3_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/win.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="star2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/win.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="star2_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/win.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="star1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/win.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="string">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>501</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 48pt &quot;AR DARLING&quot;;
font: 72pt &quot;AR DARLING&quot;;</string>
   </property>
   <property name="text">
    <string>Perfect!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>460</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;AR CHRISTY&quot;;</string>
   </property>
   <property name="text">
    <string>score : </string>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 32pt &quot;AR CHRISTY&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>550</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
font: 28pt &quot;Aldhabi&quot;;</string>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
